Source/Fuse.Nodes/Visual.BeginRemove.uno

```diff
@@ -84,7 +84,7 @@ namespace Fuse
 			}
 			else
 			{
-				args.Remove();				
+				UpdateManager.AddDeferredAction(args.Remove);
```

Comment:
We were able to do this correct on release-1.1 without having to change the tests. Why can't we get this working here?